tests/eventbus_test.py

```diff
@@ -48,7 +75,9 @@ def test_has_event(self):
         self.eventbus.event_log['foo'] = 'bar'
         assert self.eventbus.has_event('foo')
 
-    def test_sync_load_log(self):
+    @mock.patch('tron.eventbus.time', autospec=None)
```

Comment:
why not autospec=True?

Reply:
this is actually cargo-culted, i don't even know what autospec does and why we need it :) i didn't notice any difference between having autospec=True vs autospec=None.

Reply:
I just spent a couple of minutes to read the history of cargo cult. TIL

Reply:
autospec tries to prevent you from making method calls, etc that don't exist on the object you're mocking, it's usually a good idea :)